src/mlpack/methods/ann/loss_functions/huber_loss.hpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file huber_loss.hpp
+ * @author Mrityunjay Tripathi
+ *
+ * Definition of the Huber Loss function.
```

Comment:
I guess, to be correct we should go with `Definition of the Huber loss function`.